netdata.spec.in

```diff
@@ -433,6 +433,7 @@ rm -rf "${RPM_BUILD_ROOT}"
 %defattr(0755,netdata,netdata,0755)
 %{_libexecdir}/%{name}
 %{_sbindir}/%{name}
+%{_sbindir}/%{name}cli
```

Comment:
Please do not install /usr/sbin/netdatacli with netdata:netdata ownership, as we will probably have to change it later. There has been a lot of effort to change the permissions away from using netdata:netdata as much as possible. Check #6619 and #7244.

I started implementing @stormi's suggestions but hadn't had the chance to finish it.

Reply:
Also, I would do:

```suggestion
%{_sbindir}/netdatacli
```

but this is only personal preference and it is up to you.

Reply:
Good point on the permissions, I had forgotten about that completely.

Should I just fix up the permissions for the main netdata binary while I'm at it?  They should probably be `root:root` instead of `netdata:netdata`.

Reply:
You probably shouldn't change the main netdata binary permissions with this PR. Because then the permissions would be inconsistent with the other distribution channels (i.e.: kickstart.sh, kickstart-static64.sh, DEB packages, etc)

I would do all of them in one PR, in #7301

Reply:
@Ferroin, also consider something like:

```suggestion
%attr(0755,root,root) %{_sbindir}/netdatacli
```

in the future. The is no point using %defattr for only one file.